timemachine/fe/protocol_refinement.py

```diff
@@ -0,0 +1,30 @@
+from typing import Callable, List, Optional, TypeVar
+
+_T = TypeVar("_T")
+
+
+def greedy_bisection_step(
+    protocol: List[_T],
+    local_cost: Callable[[_T, _T], float],
+    make_intermediate: Callable[[_T, _T], _T],
+    callback: Optional[Callable[[float, _T, _T, _T], None]] = None,
+) -> List[_T]:
```

Comment:
Documentation request: docstring commenting on possible choices for local_cost and make_intermediate

Reply:
Added in https://github.com/proteneer/timemachine/pull/959/commits/bf7346f12455fe4e2a39767d6b45185fa7fb956e